src/webapis/WebAPIs.ts

```diff
@@ -56,6 +58,21 @@ export class WebAPIs {
             });
             console.log(`Configuring ${API_PREFIX}/report/:room_id/:event_id... DONE`);
         }
+
+        // Configure ruleserver API.
+        // FIXME: Doesn't this need some kind of access control?
```

Comment:
Good idea. We should probably let admins optionally hardcode some kind of simple HTTP auth password in the configuration. If necessary, we can make this stronger (and/or configurable with @jesopo 's patch?) later.